### PR TITLE
Implemented Submit button and refresh button to use api

### PR DESCRIPTION
**Pull request by stephnna**
Hi! code reviewer,

I implemented Submit button and refresh button such that when a user clicks submit button, the user's data, gets added to the API and when the user clicks the refresh button, the added data displays on DOM.
Kindly review.
Thanks!

### CODE REVIEW

File: src/modules/leader-class.js [hunk 1]
```diff
@@ -1,63 +1,69 @@
...
-      this.awesomeBooks.style.display = 'none';
+  // get scores from api
+getScores = async () => {
+  const response = await fetch(this.url);
+  const { result } = await response.json().catch((error) => new Error(error));
+  this.displayScores(result);
+  const scoreContent = document.querySelectorAll('.score-container');
+  scoreContent.forEach((item, index) => {
+    if (index % 2 !== 0) {
+      document.getElementById(item.id).style.backgroundColor = '#D0D0D0';
```
Comment (sja-thedude) [inline src/modules/leader-class.js]: - [x] When I try to enter user and scores it is not displayed on the leaderboard, Kindly make sure to fix it.

https://user-images.githubusercontent.com/70269507/180040077-3caab03c-e3ab-4453-b4d6-01e14dff5e49.mov

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @stephnna, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Correct Gitflow
✔️ All linters passing

## Required 
- [x] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/leader-class.js (hunk 1 above)
Comment (stephnna) [reply src/modules/leader-class.js]: Fixed
Thanks!

**Review by FrederickMih — APPROVED** [review]
# Status: Approved :tada: :tada: 

Hi @stephnna , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**